### DISCUSSION TITLE
Question about vcpkg folders

### DISCUSSION THREAD

**Discussion by kotori2** [microsoft/vcpkg#30646] [Q&A] [answered]
I'm working on adding new ports to vcpkg.
I saw there is `installed` and `packages`, one contains all installed package while the other includes individual port. I'm wondering what is the difference between them, and which should I use when I'm specifying include/library path?

**Comment by autoantwort** [comment, marked as answer]
You should use `installed`, `packages` is only the internal staging area.

**Reply by kotori2** [reply]
Thank you!
